plugins/modules/ovirt_host_storage_info.py

```diff
@@ -134,26 +142,20 @@ def main():
         # Get Host
         hosts_service = connection.system_service().hosts_service()
         host_id = get_id_by_name(hosts_service, module.params['host'])
-        storage_type = _get_storage_type(module.params)
         host_service = hosts_service.host_service(host_id)
 
-        if storage_type == 'iscsi':
+        if module.params.get('iscsi'):
             # Login
-            iscsi = module.params.get('iscsi')
-            _login(host_service, iscsi)
+            _login(host_service, module.params.get('iscsi'))
 
         # Get LUNs exposed from the specified target
         host_storages = host_service.storage_service().list()
-
-        if storage_type == 'iscsi':
-            filterred_host_storages = [host_storage for host_storage in host_storages
-                                       if host_storage.type == otypes.StorageType.ISCSI]
-            if 'target' in iscsi:
-                filterred_host_storages = [host_storage for host_storage in filterred_host_storages
-                                           if iscsi.get('target') == host_storage.logical_units[0].target]
-        elif storage_type == 'fcp':
-            filterred_host_storages = [host_storage for host_storage in host_storages
-                                       if host_storage.type == otypes.StorageType.FCP]
+        if module.params.get('iscsi') is not None:
+            host_storages = list(filter(lambda x: x.type == otypes.StorageType.ISCSI, host_storages))
```

Comment:
I less like such names as "x" (would prefer "host_storage" instead), but it looks like this is a convention :)

Reply:
Yeah, it's a convention, we use often x or if there is some short name for the var.